### ISSUE TITLE
Make it work for normal headphone remotes

### ISSUE THREAD

**Issue by cfriedrichh** [jguice/mac-bt-headset-fix#31]
The functionality of this application is very welcome! Would it be possible, though, to extend the functionality by having exactly the same behaviour as now also for regular, wired headphones? Currently, these also only work with iTunes and no other apps.

**Comment by resure** [comment]
:+1: 

Also, current version of this tool disables ability to use volume controls on apple headphones.

**Comment by jguice** [comment]
@resure I've just released a new beta (macOS 10.12.1+ only) that doesn't mess with any existing media controls and should fix that volume control issue.

@cfriedrichh What headphones are you using?  I'll give it a shot in the new version. 😃

**Comment by resure** [comment]
Don't have bt headset with me to check it on Sierra, but indeed volume controls on earpods now work fine :)

About @cfriedrichh issue:
1. Connect standard earpods to mac
2. Launch any media player that can be controlled with media buttons on keyboard (spotify, chrome with google play extension, etc.)
3. You can control that apps with media buttons on keyboard, but middle-button clicks on earpods are still getting passed directly for iTunes. It would be really nice to use that button to control any media player in the system (one click for pause, two clicks for next track, three clicks for previous one).

**Comment by jguice** [comment]
Let me know if this is still broken in 10.12.2 (if you're using Sierra).  Thanks!

**Comment by resure** [comment]
Yes, it still works finely with BT headsets on Sierra, but passes clicks from wired headsets to iTunes.
